rxandroidble/src/test/groovy/com/polidea/rxandroidble2/scan/ScanFilterTest.groovy

```diff
@@ -61,6 +58,7 @@ class ScanFilterTest extends Specification {
         !objectUnderTest.matches(mockInternalScanResult)
     }
 
+    @Ignore
```

Comment:
Do you know why ParcelUuid is returning null here? It's like it's been globally mocked...

Reply:
`ParcelUuid` is an `android` class. Tests (unless run on Android device/simulator) do not have this class available. This is the same class of problem that makes testing Android `Activity` difficult.

Reply:
Ah. Why does it work in the mock tests? Is there a way to get it working here?

Reply:
Here you try to instantiate a class that is not available at runtime. If you would create a mock of it — it should work

Reply:
But ParcelUuid is not mocked in the mock tests? Is it anything to do with `@Config(manifest = Config.NONE, constants = BuildConfig, sdk = Build.VERSION_CODES.LOLLIPOP)` which is used on the mock test class but not on the ScanFilter test class? I tried adding it but it can't find `BuildConfig`

Reply:
Ah yes that did it, just had to add `import com.polidea.rxandroidble2.BuildConfig`

Reply:
No it didn't actually. Still confused.

Reply:
Got it, just had to extend `ElectricSpecification`